modules/afmongodb/afmongodb-legacy-grammar.c

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) 2010-2016 Balabit
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 as published
+ * by the Free Software Foundation, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * As an additional exemption you are allowed to compile & link against the
+ * OpenSSL libraries as published by the OpenSSL project. See the file
+ * COPYING for details.
+ *
+ */
+
+#include "afmongodb-legacy-grammar.h"
+#include "afmongodb-legacy-private.h"
+
+gboolean
+afmongodb_dd_check_address(LogDriver *d, gboolean local)
```

Comment:
Do you have a recommendation? Would the following pair do it?
* `afmongodb_dd_validate_socket_combination`
* `afmongodb_dd_validate_network_combination`